includes/class-sensei-analysis.php

```diff
@@ -282,24 +282,52 @@ public function analysis_page() {
 		// phpcs:ignore WordPress.Security.NonceVerification.Recommended
 		$type = isset( $_GET['view'] ) ? sanitize_key( wp_unslash( $_GET['view'] ) ) : 'students';
 
+		// Set up default properties for logging an event.
+		$event_properties = [ 'view' => '' ];
+
 		if ( 0 < $lesson_id ) {
 			// Viewing a specific Lesson and all its Learners
 			$this->analysis_lesson_users_view( $lesson_id );
-		} elseif ( 0 < $course_id && ! $user_id && 'user' == $type ) {
+
+			$event_properties['view'] = 'course-lesson-users';
```

Comment:
@donnapep FYI, I've corrected the value of this property from `lesson-learners` to `course-lesson-users` so it's consistent with the [reports download view value](https://github.com/Automattic/sensei/blob/7138226ee94a86b8683b2564af01efff0167de47/includes/class-sensei-analysis.php#L758).